lib/web_ui/lib/src/engine/js_interop/js_loader.dart

```diff
@@ -0,0 +1,94 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@JS()
+library js_loader;
+
+import 'package:js/js.dart';
+
+import 'js_promise.dart';
+
+/// Typedef for the function that notifies JS that the main entrypoint is up and running.
+/// As a parameter, a [FlutterEngineInitializer] instance is passed to JS, so the
+/// programmer can control the initialization sequence.
+typedef DidLoadMainDartJsFn = void Function(FlutterEngineInitializer);
+
+/// A hook to notify JavaScript that Flutter is up and running!
+/// This is setup by flutter.js when the main entrypoint bundle is injected.
+@JS('_flutter.loader.didLoadMainDartJs')
+external DidLoadMainDartJsFn? get didLoadMainDartJs;
```

Comment:
A way to avoid `try/catch` to check for presence of `_flutter`.

```
// _flutter?.loader?.didLoadMainDartJs != null;

@JS('_flutter')
external FlutterNamespace? get _flutter;

@JS()
@anonymous
class FlutterNamespace {
  external FlutterLoaderNamespace? get loader;
}

@JS()
@anonymous
class FlutterLoaderNamespace {
  external DidLoadMainDartJsFn? get didLoadMainDartJs;
}
```

Reply:
Giving this a shot